happiness-backend/tests/test_journals.py

```diff
@@ -36,27 +37,41 @@ def auth_key_header(token, key):
 def test_e2e(init_client):
```

Comment:
Is it possible to separate this out into more tests? This will make the code more readable and make it easier to skip tests when necessary.